Upstream: No
Reason: Skip test which fails when user can bind to privileged ports,
see: https://github.com/mitmproxy/mitmproxy/issues/2730#issuecomment-354542118

diff --git a/test/mitmproxy/test_proxy.py b/test/mitmproxy/test_proxy.py
index 6c82359b9..eb5f13d7e 100644
--- a/test/mitmproxy/test_proxy.py
+++ b/test/mitmproxy/test_proxy.py
@@ -52,6 +53,7 @@ class TestProxyServer:
 
     @skip_windows
     @pytest.mark.skipif(platform.system() != "Linux", reason="Linux-only")
+    @pytest.mark.skip(reason="CI can bind to privileged ports which makes this test fail")
     def test_err(self):
         # binding to 0.0.0.0:1 works without special permissions on Windows and macOS Mojave+
         conf = ProxyConfig(options.Options(listen_port=1))
